### PR TITLE
Add cfSite GCM forcing artifact

### PR DESCRIPTION
**Pull request by Julians42**
Checklist:
- [x] I created a new folder `$artifact_name`
  - [x] I added a `README.md` in that that folder that
    - [x] describes the data and processing done to it
    - [x] lists the sources of the raw data
    - [ ] lists the required citation, licenses
  - [ ] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [x] I added the scripts that retrieve, process, and produce the artifact
  - [x] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [x] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [x] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [x] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

File: cfsite_gcm_forcing/Project.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[deps]
+ClimaArtifactsHelper = "6ffa2572-8378-4377-82eb-ea11db28b255"
```
Comment (Sbozzolo) [inline cfsite_gcm_forcing/Project.toml:2]: There's a missing newline to this line

File: cfsite_gcm_forcing/create_artifact.jl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+using Downloads 
+
+using ClimaArtifactsHelper
+
+
```
Comment (Sbozzolo) [inline cfsite_gcm_forcing/create_artifact.jl:5]:
```suggestion

```

**Review by Sbozzolo — APPROVED** [review]
Awesome, thank you!

File: cfsite_gcm_forcing/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<p align="center">
+  <img src="shen2022_f1.png" alt="Description of Image" width="500" height=auto> 
+
+  Figure from [Shen et al., 2022](https://doi.org/10.1029/2021MS002631) depicting the locations of the 21 cfSites contained in this dataset. 
+</p>
+
+The dataset contains the following time and height varying variables at each location. Please refer to the standard CMIP naming conventions [here](https://airtable.com/appYNLuWqAgzLbhSq/shrKcLEdssxb8Yvcp/tblL7dJkC3vl5zQLb) for more details on each variable. Variables below are measured at each of the 38 heights, and include atmospheric temperature and potential temperature (`ta`, `alpha`), windspeed and direction (`ua`, `va`, `wap`), and cloud properties (`cl`, `cli`, $\dotsc$):
```
Comment (szy21) [inline cfsite_gcm_forcing/README.md:10]:
```suggestion
The dataset contains the following time and height varying variables at each location. Please refer to the standard CMIP naming conventions [here](https://airtable.com/appYNLuWqAgzLbhSq/shrKcLEdssxb8Yvcp/tblL7dJkC3vl5zQLb) for more details on each variable. Variables below are measured at each of the 38 heights, and include the atmospheric state and large-scale forcing:
```

File: cfsite_gcm_forcing/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+```julia
+ps, ts, rsdt, hfls, hfss, pr, rsut, rsutcs, rlut, rlutcs, clwvi, clivi, lwp3000
+```
+
+As well as static variables: 
+```julia
+label, coszen, gustiness, windstd, windrms
+```
+
+License: Creative Commons Attribution 4.0 International
```
Comment (szy21) [inline cfsite_gcm_forcing/README.md:26]:
```suggestion
License: Creative Commons Attribution-Noncommercial-Noderivatives 4.0 International
```

**Pull request merged**